examples/chainer/chainer_integration.py

```diff
@@ -23,13 +21,6 @@
 from optuna.trial import TrialState
 
 
-# TODO(crcrpar): Remove the below three lines once everything is ok.
-# Register a global custom opener to avoid HTTP Error 403: Forbidden when downloading MNIST.
-opener = urllib.request.build_opener()
-opener.addheaders = [("User-agent", "Mozilla/5.0")]
-urllib.request.install_opener(opener)
-
-
```

Comment:
I'm not sure if we can remove the UA of the HTTP requests to download MNIST in the example of non-pytorch examples including Chainer. Mayb we should discuss in https://github.com/optuna/optuna/issues/2469.

Reply:
Thanks, you're absolutely right. I omitted the removals of these UA configurations.